index.html

```diff
@@ -1265,7 +1165,61 @@ <h4 id="example-get-all-patient-types">Example: Create Payment Response</h4>
     transactionStatusId : 6,
     authorizedAmount : '132.00',
     creditCardBrand : 'visa', 
-    id : 1,
+    id : 97DD69AA-C036-4D13-AA52-004BC25DC7A7,
+    creditCardLast4 : '1234'
+},  
+            </pre>
+            <p>&nbsp;</p>
+            <!-- Cancel Payment -->
+            <h1 id='cancel-payment'>Cancel Payment</h1>
+
+            <h2 id='cancel-payment-2'>Cancel Payment</h2>
+            <h4 id="example-cancel-payment-endpoint">Endpoint</h4>
+            <p><code>DELETE https://payfac.nextech.com/api/public/payments/{id}</code></p>
+            <h4 id="example-cancel-payment-endpoint-2">Request Parameters</h4>
+            <table>
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Description</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>id</td>
+                        <td>The Id for the payment intended to be canceled</td>
+                        <td>Guid</td>
+                    </tr>
+                </tbody>
+            </table>
+            <h4>Response Object</h4>
+            <p><a href="#payment-dto">PaymentDTO</a></p>
```

Comment:
change PaymentDTO for PaymentFullDTO

`<a href="#payment-full-dto">List of PaymentFullDTO</a>`

Reply:
done